src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java

```diff
@@ -170,30 +170,32 @@ private static void verifyInputFile(Map<String, List<String>> allTests, File fil
 
     private static void verifyInputFile(Map<String, List<String>> allTests, boolean skipFileNaming,
             String path, String fileName) {
-        List<String> classes;
-        int slash = path.lastIndexOf(File.separatorChar);
-        String packge = path.substring(0, slash);
-        boolean found = false;
-
-        for (int depth = 0; depth < 4; depth++) {
-            // -@cs[MoveVariableInsideIf] assignment value is modified later so it can't be
-            // moved
-            final String folderPath = packge;
-            slash = packge.lastIndexOf(File.separatorChar);
-            packge = path.substring(0, slash);
-            classes = allTests.get(packge);
-
-            if (classes != null
-                    && checkInputMatchCorrectFileStructure(classes, folderPath, skipFileNaming,
-                            fileName)) {
-                found = true;
-                break;
+        if (!skipFileNaming) {
+            List<String> classes;
```

Comment:
this change is not logical, after removed it, all works as expected and no errors raised.

Reply:
Maybe because you are not running MacOS? It gives errors for me.